model/patch/db.go

```diff
@@ -305,7 +305,7 @@ func PatchesByProject(projectId string, ts time.Time, limit int) db.Q {
 	}).Sort([]string{"-" + CreateTimeKey}).Limit(limit)
 }
 
-// FindFailedCommitQueuePatchesInTimeRange returns failed patches if they started within range,
+// FindFailedCommitQueuePatchesinTimeRange returns failed patches if they started within range,
```

Comment:
I'm not sure how you found this haha but it's actually the function name, not the comment, that's incorrect if you wouldn't mind fixing that